tests/e2e_tests/bp_map_e2e.py

```diff
@@ -0,0 +1,327 @@
+import argparse
+import os
+import pytest
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib import colors
+from astropy.io import fits
+from corgidrp import data
+from corgidrp import caldb
+from corgidrp import detector
+from corgidrp import darks
+from corgidrp import walker
+
+# Get the directory of the current script file
+thisfile_dir = os.path.dirname(__file__)
+
+@pytest.mark.e2e
+def bp_map_master_dark_e2e(tvacdata_path, e2eoutput_path):
```

Comment:
One last thing: based on how conftest.py collects tests for automated testing, the two tests here don't get identified unless the functions start with `test_*`. We just need to prepend "test_" to each function name.

Reply:
done